src/main/java/com/redhat/documentation/asciidoc/cli/ExtractionRunner.java

```diff
@@ -165,6 +167,29 @@ private void writeModules(Configuration config) {
         }
     }
 
+    private void moveNonadoc(Configuration config) {
+        try{
+        Path assetsDir = Files.createDirectories(config.getOutputDirectory().toPath().resolve("assets"));
+        final String extension = ".asciidoc";
+        final File sourceDir = config.getSourceDirectory();
+        final File destinationDir = assetsDir.toFile();
+        File[] files = sourceDir.listFiles((File pathname) -> !(pathname.getName().endsWith(extension)));
```

Comment:
This won't work for the other asciidoc file extensions. 

.asciidoc, .adoc, and .ad are all valid extensions.

Take a look at https://github.com/asciidoctor/asciidoctorj/blob/master/asciidoctorj-core/src/main/java/org/asciidoctor/jruby/AsciiDocDirectoryWalker.java for the regex they're using to find asciidoc files.